device/globals/action_menu.gd

```diff
@@ -31,20 +27,13 @@ func _input(event):
 	if !event.is_pressed():
 		return
 
-	for a in actions:
-		if event.is_action(a.get_name()):
-			action_pressed(a.get_name())
-			break
-
 func _ready():
 
-	actions = []
 	var acts = get_node("actions")
 	for i in range(acts.get_child_count()):
 		var c = acts.get_child(i)
 		if !c.is_type("Button"):
```

Comment:
I think this needs to be `if !c is Button:` in Godot 3.0. Does the editor not complain about this?